src/lib.rs

```diff
@@ -44,6 +47,36 @@ pub mod web {
     };
 }
 
+// FIXME Remove this in the future
+pub(crate) const fn new_btreemap<K: ?const Ord, V>() -> BTreeMap<K, V> {
+    use std::mem::{forget, transmute};
+    use std::cmp::Ordering;
+    #[derive(PartialEq, Eq, PartialOrd)]
+    struct ConstOrdWrapper<T>(T);
```

Comment:
```suggestion
    #[transparent]
    struct ConstOrdWrapper<T>(T);
```

otherwise the transmute is unsound